docs/usage_with_relay.md

```diff
@@ -32,7 +32,7 @@ Modify your `relay.config.js` file to reflect the following:
    ```
 
    - `schemaConfig` tells the Relay compiler where to find the `nodeId` field on the `node` interface
-   - `customScalars` will improve Relay's type emission
+   - `customScalarTypes` will improve Relay's type emission
```

Comment:
@nesskazu could you please add a note that in relay versions older than v16.2.0 it should be named `customScalars`.

Reply:
Thanks for the feedback. I've added the note. https://github.com/supabase/pg_graphql/pull/547/commits/74368fc4b1659b5e099e40271647f78c30bc7cb7